lib/iris/plot.py

```diff
@@ -844,7 +844,9 @@ def _replace_axes_with_cartopy_axes(cartopy_proj):
                 ylabel=ax.get_ylabel(),
             )
         else:
+            position = ax.get_position()
```

Comment:
You may want the original position?
```suggestion
            position = ax.get_position(original=True)
```

Reply:
Thanks @QuLogic!